<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProyectoInstrumentosDialog</class>
 <widget class="QewTreeViewDialog" name="ProyectoInstrumentosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ewTreeViewDialog</string>
  </property>
  <property name="containerName">
   <cstring>stackedWidget</cstring>
  </property>
  <property name="treeNavigatorName">
   <cstring>qewListViewInst</cstring>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="qewListViewInst">
      <column>
       <property name="text">
        <string>Instrumentos</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Nuevo instrumento</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="datosProyecto">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="idInstLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Descripci√≥n:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="descriptionTextEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QewTreeViewDialog</class>
   <extends>QewExtensibleDialog</extends>
   <header>qewtreeviewdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qewListViewInst</sender>
   <signal>itemPressed(QTreeWidgetItem*,int)</signal>
   <receiver>ProyectoInstrumentosDialog</receiver>
   <slot>setCurrentWidget(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
